docs/development_suite/webhooks-and-events/events.mdx

```diff
@@ -100,21 +333,103 @@ hence there is no guarantee that the service will be found in the project config
 | `repositoryURL` | `string` | The URL of the repository that hosts the service code                                        |
 | `serviceName`   | `string` | The name of the service inside the project configuration                                     |
 | `templateId`    | `string` | The identifier of the template in the Console marketplace from which the service was created |
+| `service`       | `object` | The service metadata (ideally, the whole service resource)                                   |
 
 <Tabs>
 <TabItem value="example" label="Example" default>
 
 ```json
-  {
-    "eventName": "service_created",
-    "eventTimestamp": 123456,
-    "payload": {
-      "tenantId": "my-company-id",
-      "projectId": "0000000000000001",
-      "repositoryURL": "http://gihub.com/my-repository",
-      "serviceName": "my-service",
-      "templateId": "0000000000000002"
+{
+  "eventName": "service_created",
+  "eventTimestamp": 123456,
+  "payload": {
+    "tenantId": "my-company-id",
+    "projectId": "0000000000000001",
+    "repositoryURL": "http://gihub.com/my-repository",
+    "serviceName": "my-service",
+    "templateId": "0000000000000002",
+    "service": {
+      "serviceName": "my-template",
+      "dockerImage": "docker-hub/my-template",
+      "repoId": 12345,
+      "webUrl": "https://git.host/folder/company/project/services/my-template",
+      "sshUrl": "git@git.eu:folder/company/project/services/my-template.git",
```

Comment:
There are both? This info are not of the new service instead of the original template?
Or we can set both the information, the template and tthe generated service